website/content/en/preview/tasks/amitasks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Managing AMIs"
+linkTitle: "Managing AMIs"
+weight: 10
+description: >
+  Tasks for managing AMIS in Karpenter
+---
+
+Understanding how Karpenter assigns AMIs to nodes can help ensure that your workloads will run successfully on those nodes and continue to run if the nodes are upgraded to newer AMIs.
+Below we describe how Karpenter assigns AMIs to nodes when they are first deployed and how newer AMIs are assigned later when nodes are spun up to replace old ones.
+Later, there are tasks that describe the ways that you can intervene to assert control over how AMIs are used by Karpenter for your clusters.
+
+Features for managing AMIs described here should be considered as part of the larger upgrade policies that you have for your clusters.
+See [How do I upgrade an EKS Cluster with Karpenter]({{< relref "../faq/#how-do-i-upgrade-an-eks-cluster-with-karpenter" >}}) for details on this process. 
+
+## How Karpenter assigns AMIs to nodes by default
+
+If you do nothing to modify how Karpenter handles AMIs, here is how Karpenter assigns AMIs nodes:
+
+* The default AMI family used by Karpenter is `amiFamily: AL2`. 
```

Comment:
Thanks. I'll say this instead:
When you create an `EC2NodeClass`, you are required to set the family of AMIs to use. For example, for the AL2 family, you would set `amiFamily: AL2`.
* With that `amiFamily` set, any time Karpenter needed to spin up a new node, it would use the latest in the AL2 family.